packages/cli/src/rolldown/config.ts

```diff
@@ -76,8 +76,34 @@ export function getRolldownConfig(apibara: Apibara): RolldownOptions {
     },
   );
 
+  rolldownConfig.plugins?.push(
+    replace({
+      preventAssignment: true,
+      values: {
+        "process.env.APIBARA_CONFIG": getSerializedRuntimeConfig(apibara),
+      },
+    }) as RolldownPluginOption,
+  );
   rolldownConfig.plugins?.push(indexers(apibara));
   rolldownConfig.plugins?.push(appConfig(apibara));
 
   return rolldownConfig;
 }
+
+function getSerializedRuntimeConfig(apibara: Apibara) {
+  try {
+    return JSON.stringify(
```

Comment:
Should we use a version of this that support bigint or not?

Reply:
I think bigint is useful for example for starting block so it makes sense. We should have the helpers for it somewhere since it's used by the vcr code too.